README.md

```diff
@@ -1,88 +1,69 @@
-## Code of Conduct
+## Folder system
 
-* All work must include a README with the following elements:
+    ├── client  
+    └── server (this works containerized)                    
+        ├── /api           
+        ├── /db         
+        └── docker-compose.yaml
 
-    * Must have:
-        * Assignment title
-        * Installation & usage
-        * Changelog
-        * Bugs
-
-    * Could have:
-        * Wins & Challenges
-
-## Useful Tools
-
-* [Markdown Guide](https://guides.github.com/features/mastering-markdown/)
-
-## Example README
-
-Below is an example of what a README could look like.
-
-# Assignment 1
 
 ## Installation & Usage
 
 ### Installation
 
-* Clone or download the repo.
-* Open terminal and navigate to `fizzbuzz` folder.
-* Run `npm install` to install dependencies.
+* Clone the repo.
+* Open terminal and navigate to `server` folder.
+* Run `docker compose up` to initialize containers.
 
 ### Usage
 
-* Run `npm start` to launch server.
-* Run `npm test` to launch test suite.
+* Run `http-server` to launch the client on port 8080.
+
+# CONTENT
 
-# Routes
+## Server routes
 
-### Routes
 | **URL** | **HTTP Verb** |  **Action**|
 |------------|-------------|------------|
-| /thoughts/         | GET       | index  
-| /thoughts/new      | GET       | new   ??
-| /thoughts          | POST      | create   
-| /thoughts/:id      | GET       | show  
-
-# Server
+| /posts/    | GET         | index  
+| /posts     | POST        | create   
+| /posts/:id | GET         | show  
 
-## Posts
-| id | title |  body | user_id|
-|----|-------|-------|------- |
-|  1 |title1 | body1 | 1      |
-|  2 |title2 | body2 | 1      |
-|  3 |title3 | body3 | 2      |
 
 
-## Users
-| id  | pseudonym |
-|-----|---------- |
-| 1   | name1     |
-| 2   | name2     |
+## Posts database structure
+| id | title |  main | user_id|
+|----|-------|-------|-------    |
+|  1 |title1 | body1 | username1 |
+|  2 |title2 | body2 | username2 |
+|  3 |title3 | body3 | username3 |
 
 
 
 ## Changelog
 
-### index.html
+### client
+
+[x] Removed filepaths to server side 
+
+[x] Add port nuber directly
 
-[x] Fixed filepath to correctly connect CSS to HTML file.
+[x] Added favicon
 
-### server.js
+### server
 
-[x] Updated `fizzbuzz` funtion to accept an arguement.
+[x] Removed users table
 
 ## Bugs
 
-[] Function logs 15 as `fizz` when it should log `fizzbuzz`
+[] No route to retrieve messages on client side
```

Comment:
There is a hack with `npm http-server` where you can use 404.html as the default web page to make a single page application